manon/accordion.js

```diff
@@ -45,7 +45,7 @@ function initAccordion(accordion) {
     }
 
     // Set up the initial `aria-expanded` state.
-    if (button.getAttribute("aria-expanded")) {
+    if (button.getAttribute("aria-expanded") === "true") {
```

Comment:
Does this break the use case of explicitly setting `aria-expanded="false"` on the first button in the markup to stop it from being expanded by default?

Reply:
It should not,  and is more explicit, only an element with aria-expanded="true" should now be opened on page load.

Reply:
PR: https://github.com/minvws/nl-rdo-manon/pull/636